### DISCUSSION TITLE
Is Pybricks Compatible with BuWizz Hubs/Motors?

### DISCUSSION THREAD

**Discussion by RyanEdwardHall** [pybricks/support#549] [Q&A] [answered]
Hi Everyone, 
I'd really like to get a hub with a rechargeable battery pack, and it seems like the hubs and motors by [BuWizz](https://buwizz.com/shop/buwizz-3-0-pro/) are pretty good. I'm hoping to find out if they're compatible with the pybricks project. If not, are there any community-recommended alternatives? Any idea how much work it might be to add support? If I pursue the idea, I'm not afraid to roll up my sleeves and send in a pull request :smile: 
Thanks!

**Comment by laurensvalk** [comment, marked as answer]
We support only LEGO hubs, motors, and sensors at this time, mainly for safety/legal reasons.

But we're always happy to discuss with third-party hardware manufacturers to see what's possible. (They can reach us at `team@pybricks.com`.)

**Comment by ZPhilo** [comment]
The motor itself should work (since it has no encoder, only DCMotor() can be used). But... the motor driver chip inside LEGO hubs is by far too puny to properly drive the BuWizz motor!
